src/interfaces/coral_web/src/components/Configuration/Settings.tsx

```diff
@@ -38,6 +40,7 @@ export const Settings: React.FC = () => {
 
   return (
     <div className="flex flex-col gap-y-6 px-5 pb-10">
+      {isExperiementalFeaturesOn && <ExperimentalModeOverlay />}
```

Comment:
correct me if im wrong but we could just delete this overlay and people would be able to interact with the elements below right? 🤣 maybe we just replace this with a ternary statement and the below component (without the `absolute` positioning)

Reply:
yeah that's correct haha-- i thought it'd be nice to be able to see what you have access to with experimental mode off but your suggestion is more hack proof